Tasks/KubernetesV0/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 0,
         "Minor": 1,
-        "Patch": 30 
+        "Patch": 30
     },
     "demands": [],
     "preview": "false",
```

Comment:
Have to check in the task.loc.json generated for KubernetesV0 task

Reply:
There is no patch version change actually, I think someone has already bumped up the V0 task version here to 30, which is what Harshil's PR was doing. This change is showing up due to a difference in whitespace. task.loc.json is already on 30 and no whitespace issues.